spec/lib/submission_builder/ty2024/states/md/documents/md502_spec.rb

```diff
@@ -446,5 +446,67 @@
         expect(xml.at("Form502 TaxWithheld")&.text).to eq('500')
       end
     end
+
+    context "AuthToDirectDepositInd" do
+      before do
+        intake.bank_authorization_confirmed = "yes"
+      end
+
+      it 'outputs the total state and local tax withheld' do
+        expect(xml.at("Form502 AuthToDirectDepositInd")&.text).to eq('X')
+      end
+    end
+
+    context "Line 51d: NameOnBankAccount" do
+      before do
+        intake.payment_or_deposit_type = "direct_deposit"
+        intake.account_holder_first_name = "Jack"
+        intake.account_holder_middle_initial = "D"
+        intake.account_holder_last_name = "Hansel"
```

Comment:
same question. idk if i just completely missed this on the first review sorry

Reply:
<img width="1303" alt="Screenshot 2024-11-25 at 7 39 20 PM" src="https://github.com/user-attachments/assets/c63a5f7c-054b-4f18-a460-e9880d964a68">

it works, it seems